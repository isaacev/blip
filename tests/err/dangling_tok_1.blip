let x = 123 in x y
