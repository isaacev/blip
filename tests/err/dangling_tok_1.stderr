
  ERROR: expected the end-of-file, found word

    : ./tests/err/dangling_tok_1.blip
  1 | let x = 123 in x y
    |                  ~


